docs/monitoring/elasticsearch-details.asciidoc

```diff
@@ -122,12 +137,33 @@ follower index is lagging behind the leader index, the last fetch time, the
 number of operations synced, and error messages. If you select a follower index,
 you can view the same information for each shard. For example:
 
+[role="screenshot"]
 image::monitoring/images/monitoring-ccr.png["Cross-cluster replication",link="images/monitoring-ccr.png"]
 
 If you select a shard, you can see graphs for the fetch and operation delays. 
 You can also see advanced information, which contains the results from the 
 {ref}/ccr-get-follow-stats.html[get follower stats API]. For example:
- 
+
+[role="screenshot"] 
 image::monitoring/images/monitoring-ccr-shard.png["Cross-cluster replication shard details",link="images/monitoring-ccr-shard.png"]
 
 For more information, see {stack-ov}/xpack-ccr.html[Cross-cluster replication].
+
+[float]
+[[logs-monitor-page]]
+==== Logs
+
+If you use {filebeat} to collect log data from your cluster, you can see its
+recent logs in the *Stack Monitoring* application. The *Clusters* page lists the
+number of informational, debug, and warning messages in the server and
+deprecation logs. 
+
+If you click *Logs*, you can see the most recent logs for the cluster. For
+example:
+
+[role="screenshot"]
+image::monitoring/images/monitoring-elasticsearch-logs.jpg["Recent {es} logs"]
+
+TIP: By default, up to 10 logs are shown. You can show up to 50 logs by
```

Comment:
Nit: I believe this is the number of log _lines_ and not the number of logs.

Reply:
Thanks @cachedout ! The screenshot of the "Recent logs" shows it has the following text: "Showing the most recent logs for this cluster, up to 10 total logs”.  Should that be "log entries" in the UI too?

Reply:
I'm not sure we brought this up in the original PR. Are you suggesting that `10 logs` is too ambiguous in terms of what that means? And we should add `lines` to the end to ensure it's clear to the user? Or `entries` might be a better word here? I'm not sure. Happy to go either way

Reply:
I like "entries", since it seemed like each item I saw in Kibana could contain multiple "lines" of text.

Reply:
Love it, thanks!